### PR TITLE
Add This is you indicator in collaborators table

### LINKED ISSUE #1563: Add "(This is you)" hint to user entries in collaborator tables
#### Summary
Tables that show user entries should append a `(This is you)` message next to the user id, to improve clarity.

#### Why do we need this?
Improve clarity for the user.

#### What is already there? What do you see now?
Plain display of the user id.

#### What is missing? What do you want to see?
A hint that an entry refers to the current user.

#### Environment
Browser.

#### How do you propose to implement this?
Trivial.

#### Can you do this yourself and submit a Pull Request?
@pgalic96

### PR DESCRIPTION
**Pull request by pgalic96**
#### Summary

Closes #1563 

#### Changes

- Add `This is you` indicator next to current user id 

Screenshot 2020-10-26 at 6 12 21 PM

#### Testing

- By hand

##### Regressions

none

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/console/containers/collaborators-table/index.js [hunk 1]
```diff
@@ -23,50 +24,76 @@ import FetchTable from '@console/containers/fetch-table'
...
-    render: ids => getCollaboratorId({ ids }),
-  },
-  {
-    name: 'ids',
-    displayName: sharedMessages.type,
-    render(ids) {
-      const isUser = 'user_ids' in ids
-      const icon = isUser ? 'user' : 'organization'
+  constructor(props) {
+    this.headers = [
```
Comment (bafonins) [inline pkg/webui/console/containers/collaborators-table/index.js:48]:
```
Uncaught TypeError: Cannot set property 'headers' of undefined
    at new CollaboratorsTable (index.js?b780:49)
    at new CollaboratorsTable (eval at ES6ProxyComponentFactory (react-hot-loader.development.js?c2cb:3115), <anonymous>:5:7)
    at constructClassInstance (react-dom.development.js?7f13:13309)
    at updateClassComponent (react-dom.development.js?7f13:16982)
    at beginWork$1 (react-dom.development.js?7f13:18502)
    at HTMLUnknownElement.callCallback (react-dom.development.js?7f13:348)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?7f13:398)
    at invokeGuardedCallback (react-dom.development.js?7f13:455)
    at beginWork$$1 (react-dom.development.js?7f13:23214)
    at performUnitOfWork (react-dom.development.js?7f13:22205)
```

File: pkg/webui/console/containers/collaborators-table/index.js [hunk 2]
```diff
@@ -23,50 +24,76 @@ import FetchTable from '@console/containers/fetch-table'
...
-    displayName: m.id,
-    render: ids => getCollaboratorId({ ids }),
-  },
-  {
-    name: 'ids',
-    displayName: sharedMessages.type,
-    render(ids) {
-      const isUser = 'user_ids' in ids
-      const icon = isUser ? 'user' : 'organization'
+  constructor(props) {
```
Comment (bafonins) [inline pkg/webui/console/containers/collaborators-table/index.js:46]:
```suggestion
  constructor(props) {
    super(props)
```

File: pkg/webui/console/containers/collaborators-table/index.js [hunk 3]
```diff
@@ -23,50 +24,76 @@ import FetchTable from '@console/containers/fetch-table'
...
+import { selectUserId } from '@console/store/selectors/user'
 
 import style from './collaborators-table.styl'
 
 const m = defineMessages({
   id: 'User / Organization ID',
+  currentUserIndicator: '(This is you)',
+})
+
+@connect(function(state) {
```
Comment (bafonins) [inline pkg/webui/console/containers/collaborators-table/index.js]:
```suggestion
@connect(state => {
```

**Review by bafonins — CHANGES_REQUESTED** [review]

**Comment by kschiffer** [comment]
PLEASE always add a screenshot to PRs that introduce visual changes.

File: pkg/webui/console/containers/collaborators-table/index.js [hunk 4]
```diff
@@ -23,50 +24,77 @@ import FetchTable from '@console/containers/fetch-table'
...
 const m = defineMessages({
   id: 'User / Organization ID',
+  currentUserIndicator: '(This is you)',
+})
+
+@connect(state => {
+  return {
+    currentUserId: selectUserId(state),
+  }
 })
```
Comment (kschiffer) [inline pkg/webui/console/containers/collaborators-table/index.js]:
```suggestion
@connect(state => ({
  currentUserId: selectUserId(state),
}))
```
Nit.

**Review by kschiffer — APPROVED** [review]

**Review by bafonins — APPROVED** [review]

**Pull request merged**